packages/web-twig/src/Resources/components/Header/README.md

```diff
@@ -46,126 +106,394 @@ With Html syntax lexer (enabled by default):
 </Header>
 ```
 
-### Full Header example
+## API
+
+| Prop name  | Type      | Default       | Required | Description                              |
+| ---------- | --------- | ------------- | -------- | ---------------------------------------- |
+| `class`    | `string`  | `null`        | no       | Custom CSS class                         |
+| `color`    | `string`  | `transparent` | no       | Color variant: `transparent`, `inverted` |
```

Comment:
We usually set Type to the values available. so it should be
```suggestion
| `color`    |  `transparent`, `inverted`  | `transparent` | no       | Color variant  |
```

Reply:
See Grid for example

Reply:
OK, my feeling was there is no standard way.